server/database/seed.js

```diff
@@ -8,7 +9,15 @@ const userdescriptions = ['Globetrotter', 'Avid Traveller', 'Novice Nomad'];
 
 const userages = [1, 2, 3, 4];
```

Comment:
This variable name is a little unclear to me. I noticed above that you had used a variable named `userdescriptions` as well. Consider changing to camelCase.